stock_picking_quick/models/product_product.py

```diff
@@ -0,0 +1,50 @@
+# © 2019 Today Akretion
+# @author Pierrick Brun <pierrick.brun@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    move_ids = fields.One2many(
+        comodel_name='stock.move', inverse_name='product_id',
+        help='Technical: used to compute quantities to pick.',
+    )
+
+    @api.depends('move_ids')
+    def _compute_process_qty(self):
+        res = super(ProductProduct, self)._compute_process_qty()
+        if self.env.context['parent_model'] == 'stock.picking':
+            for product in self:
+                stock_moves = self.env['stock.move'].search([
```

Comment:
@PierrickBrun no search into a loop... read_group should be your friend.

Reply:
way better now, thanks !